src/pages/pt-br/guides/deploy/aws.md

```diff
@@ -0,0 +1,234 @@
+---
+Título: Deploy seu Site Astro na AWS
+descrição: Como fazer o deploy do seu Astro Site na web usando AWS.
+layout: ~/layouts/DeployGuideLayout.astro
+i18nReady: true
+---
+
+[AWS](https://aws.amazon.com/) é uma plataforma com todos os recursos necessários para a hospedagem de aplicativos da Web que pode ser usada para implantar um site Astro.
+
+Para _Deployar_ seu projeto na AWS será necessário a utilização do [AWS console](https://aws.amazon.com/console/). (A maioria dessas ações também pode ser feitas usando o [AWS CLI](https://aws.amazon.com/cli/)). Esse guia irá lhe mostrar o passo-a-passo para deployar seu site na AWS, começando pelo método mais simples. Em seguida, demonstrará alguns passos adicionais para melhorar a eficiência de custos e desempenho. 
+
+## AWS Amplify 
+
+AWS Amplify é um conjunto de ferramentas e funcionalidades especialmente concebidas para os programadores front-end web e mobile construírem de forma rápida e fácil aplicações completas no AWS. 
+
+1. Criar um novo projeto Amplify Hosting .
+2. Conectar seu repositório ao Amplify.
+3. Modificar seu diretório de saída de _build_ de `baseDirectory` para `/dist`.
+
+    ```yaml
+    version: 1
+    frontend:
+      phases:
+        preBuild:
+          # Not using npm? Change `npm ci` to `yarn install` or `pnpm i`
+          commands:
+            - npm ci
+        build:
+          commands:
+            - npm run build
+      artifacts:
+        baseDirectory: /dist
+        files:
+          - '**/*'
+      cache:
+        paths:
+          - node_modules/**/*
+
+    ```
+
+Amplify irá _deployar_ automaticamente o seu site e o atualizará quando você _commitar_ um push no seu repositório. 
+
+## S3 static website hosting
+
+S3 é o ponto de partida para qualquer aplicação. É onde såo armazenados as pastas do seu projeto outros itens. O S3  cobra por quantidade de documentos armazenados e requisições feitas. Você pode encontrar mais informações sobre o S3 em [AWS documentation](https://aws.amazon.com/pt/s3/?nc1=h_ls).
+
+1. Crie um S3 bucket com o nome do seu projeto.
+
+    :::Dica: 
```

Comment:
Ah, same here — `tip` tells us to use some special formatting and will automatically get the correct translation from `src/i18n/pt-br/ui.ts`

```suggestion
    :::tip
```